start.py

```diff
@@ -114,11 +116,21 @@ def start():
         help="While using 'test' mode, this allows to use the default"
         " Django server instead of Uwsgi",
     )
+    parser.add_argument(
+        "--elastic",
+        required=False,
+        action="store_true",
+        help="While using 'test' mode, this spins up Elasticsearch"
```

Comment:
again, not for development only

Reply:
@mlodic You mean that the option should be available for production mode too?

Reply:
yeah, I have never said the contrary.

Reply:
Corrected it